2_govt_urls_federal_only.csv

```diff
@@ -386,4 +392,4 @@ westernmarylandcfc.org,Combined Federal Campaign,Federal Government,,accepted te
 westpoint.edu,,Federal Government,,accepted term,United States Military Academy (Army) - West Point -- Note added 1/23/14 15:26,http://www.westpoint.edu/SitePages/Home.aspx,1/23/14 15:26
 whitehousehistory.org,#N/A,Federal Government,,candidate term,The White House Historical Association -- Note added 6/6/13 18:09,#N/A,6/6/13 18:09
 yes-abroad.org,,Federal Government,,accepted term,"YES Abroad - The U.S. Department of State's Kennedy-Lugar Youth Exchange and Study Abroad (YES Abroad) program, provides competitive merit-based scholarships for U.S. citizens in secondary school (age 15-18.5) to study for one academic year in select countries. -- Note added 7/13/15 14:43",http://yes-abroad.org/,7/13/15 14:23
-yesprograms.org,,Federal Government,,accepted term,"YES Program - The Kennedy-Lugar Youth Exchange and Study (YES) program was established by Congress in October, 2002 in response to the events of September 11, 2001. The program is funded through the U.S. Department of State and sponsored by the Bureau of Educational & Cultural Affairs (ECA) to provide scholarships for high school students from countries with significant Muslim populations to spend up to one academic year in the U.S. Students live with host families, attend high school, engage in activities to learn about American society and values, acquire leadership skills, and help educate Americans about their countries and cultures. Starting in 2009, the YES Abroad program was established to provide a similar experience for U.S. students (15-18 years) to spend an academic year in select YES countries. -- Note added 7/13/15 14:22",http://www.yesprograms.org/,7/13/15 14:22
\ No newline at end of file
+yesprograms.org,,Federal Government,,accepted term,"YES Program - The Kennedy-Lugar Youth Exchange and Study (YES) program was established by Congress in October, 2002 in response to the events of September 11, 2001. The program is funded through the U.S. Department of State and sponsored by the Bureau of Educational & Cultural Affairs (ECA) to provide scholarships for high school students from countries with significant Muslim populations to spend up to one academic year in the U.S. Students live with host families, attend high school, engage in activities to learn about American society and values, acquire leadership skills, and help educate Americans about their countries and cultures. Starting in 2009, the YES Abroad program was established to provide a similar experience for U.S. students (15-18 years) to spend an academic year in select YES countries. -- Note added 7/13/15 14:22",http://www.yesprograms.org/,7/13/15 14:22

```

Comment:
Please move this entry to the quasi-governmental list. Please also add "Kennedy-Lugar Youth Exchange and Study" in column 2.